app/db/models/document/physical_document.py

```diff
@@ -0,0 +1,44 @@
+import sqlalchemy as sa
+from app.db.session import Base
+
+
+class PhysicalDocument(Base):
+    """A physical document.
+
+    Representation of a document that exists in the real world.
+    Owned and updated only by the pipeline.
+    """
+
+    __tablename__ = "physical_document"
+
+    id = sa.Column(sa.Integer, primary_key=True)
+    title = sa.Column(sa.Text, nullable=False)
+    md5_sum = sa.Column(sa.Text, nullable=True)
+    source_url = sa.Column(sa.Text, nullable=True)
+    date = sa.Column(sa.DateTime, nullable=False)
+    format = sa.Column(sa.Text, nullable=True)
+
+
+class Language(Base):
+    """Note, moved from deprecated."""
+
+    __tablename__ = "language"
+
+    id = sa.Column(sa.Integer, primary_key=True)
+    language_code = sa.Column(sa.CHAR(length=3), nullable=False, unique=True)
+    part1_code = sa.Column(sa.CHAR(length=2))
+    part2_code = sa.Column(sa.CHAR(length=3))
+    name = sa.Column(sa.Text)
+
+
+class PhysicalDocumentLanguage(Base):
+    """A document's languages."""
+
+    __tablename__ = "physical_document_language"
+
+    id = sa.Column(sa.Integer, primary_key=True)
+    language_id = sa.Column(sa.ForeignKey(Language.id), nullable=False)
+    document_id = sa.Column(
+        sa.ForeignKey(PhysicalDocument.id, ondelete="CASCADE"),
+        nullable=False,
+    )
```

Comment:
When we spoke earlier I thought we agreed to remove the int id on these join tables...?

```suggestion
    language_id = sa.Column(sa.ForeignKey(Language.id), nullable=False)
    document_id = sa.Column(
        sa.ForeignKey(PhysicalDocument.id, ondelete="CASCADE"),
        nullable=False,
    )
    
    PrimaryKeyConstraint(language_id, document_id)
```

Reply:
We can always do this in the follow-up

Reply:
FTR - this was done at the point it [was raised](https://github.com/climatepolicyradar/navigator-backend/blob/3855ca5939fa0d5a7741e01ad92c0dc0fb4c7301/app/db/models/document/physical_document.py#L34-L45) ... but closing this off now and merging 1&2 as agreed